### PR TITLE
analog: add commands, parameters and events for the 4pu multi sensor device

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: docs/analog/commands/readme.md [hunk 1]
```diff
@@ -29,38 +29,44 @@ There are `2` types of command direction:
 | `0x1f2a` | [WriteImage](./WriteImage.md#request)               | Request to write the block of the new image to the device.                   |
 | `0x1f2b` | [VerifyImage](./VerifyImage.md#request)             | Request to verify the update image on the device.                            |
 | `0x1f2c` | [UpdateRun](./UpdateRun.md#request)                 | Request to start update on the device.                                       |
+| `0x1f2c` | [UpdateRun](./UpdateRun.md#request)                 | Request to start update on the device.                                       |
```
Comment (DarkPark) [inline docs/analog/commands/readme.md:32]: duplicate

File: docs/analog/commands/readme.md [hunk 2]
```diff
@@ -29,38 +29,44 @@ There are `2` types of command direction:
...
+| `0x1e`   | [DataSegment](./DataSegment.md#response)                 | Response to the [DataSegment](./DataSegment.md#request) downlink command.                                                                         |
+| `0x1f02` | [GetLmicInfo](./GetLmicInfo.md#response)                 | Response to the [GetLmicInfo](./GetLmicInfo.md#request) downlink command.                                                                         |
+| `0x1f0a` | [ExAbsHourMC](./uplink/ExAbsHourMC.md)                   | Absolute values from device channels for the specified hour and hourly difference.                                                                |
+| `0x1f0b` | [ExAbsDayMC](./uplink/ExAbsDayMC.md)                     | Absolute values from device channels for the previous day's billing hour.                                                                         |
+| `0x1f0f` | [ExAbsCurrentMC](./GetExAbsCurrentMC.md#response)        | Response to the [GetExAbsCurrentMC](./GetExAbsCurrentMC.md#request) downlink command or current consumption absolute values from device channels. |
```
Comment (DarkPark) [inline docs/analog/commands/readme.md:61]:
```suggestion
| `0x1f0f` | [GetExAbsCurrentMC](./GetExAbsCurrentMC.md#response)        | Response to the [GetExAbsCurrentMC](./GetExAbsCurrentMC.md#request) downlink command or current consumption absolute values from device channels. |
```
?

File: docs/analog/parameter-types.md [hunk 3]
```diff
@@ -43,6 +43,7 @@ Command body structures for [SetParameter](./commands/SetParameter.md).
 * [NB-IoT APN](#nb-iot-apn)
 * [NB-IoT LED Indication](#nb-iot-led-indication)
 * [NB-IoT SIM](#nb-iot-sim)
+* [Channel type](#channel type)
```
Comment (DarkPark) [inline docs/analog/parameter-types.md:46]:
```suggestion
* [Channel type](#channel-type)
```
?

File: docs/analog/commands/readme.md (hunk 1 above)
Comment (Pnom) [reply docs/analog/commands/readme.md]: fixed

File: docs/analog/commands/readme.md [hunk 4]
```diff
@@ -29,38 +29,44 @@ There are `2` types of command direction:
 | `0x1f2a` | [WriteImage](./WriteImage.md#request)               | Request to write the block of the new image to the device.                   |
 | `0x1f2b` | [VerifyImage](./VerifyImage.md#request)             | Request to verify the update image on the device.                            |
 | `0x1f2c` | [UpdateRun](./UpdateRun.md#request)                 | Request to start update on the device.                                       |
+| `0x1f2c` | [UpdateRun](./UpdateRun.md#request)                 | Request to start update on the device.                                       |
+| `0x1f32` | [ChannelsStatus](./GetChannelsStatus.md#request)    | Command to get the status of device channels.                                |
+| `0x1f33` | [UpdateRun](./GetChannelsTypes.md#request)          | Command to get the types of channels on the device.                          |
```
[the remark this answers is not in the document]
Comment (Pnom) [reply docs/analog/commands/readme.md]: fixed

File: docs/analog/commands/readme.md (hunk 2 above)
Comment (Pnom) [reply docs/analog/commands/readme.md]: fixed

File: docs/analog/parameter-types.md (hunk 3 above)
Comment (Pnom) [reply docs/analog/parameter-types.md]: fixed

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**